runtime/list_test.go

```diff
@@ -392,22 +392,20 @@ func TestListSetItem(t *testing.T) {
 }
 
 func TestListSort(t *testing.T) {
-	fun := newBuiltinFunction("TestListSetItem", func(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
-		if raised := checkFunctionArgs(f, "TestListSetItem", args, ListType); raised != nil {
-			return nil, raised
-		}
-		l := toListUnsafe(args[0])
-		if raised := l.Sort(f); raised != nil {
+	fun := newBuiltinFunction("TestListSort", func(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
+		if _, raised := listSort(f, args, KWArgs{}); raised != nil {
```

Comment:
You're indirectly testing both. `GetAttr(f, ListType.ToObject(), NewStr("sort"), nil)` is the equivalent of `getattr(list, 'sort')`. By calling sort this way, you're invoking listSort, which is in turn calling `func (l *List) Sort()`.